### ISSUE TITLE
Note element width wants to be dynamic

### ISSUE THREAD

**Issue by aaron-em** [fdansv/marginotes#1]
![responsive oops](https://cloud.githubusercontent.com/assets/5650970/13637713/cf41fb36-e5d6-11e5-8aec-44b075e464cb.PNG)
`window.innerWidth` is 1110px. 

From poking at it, it looks like the margin note declines to display itself if it would have a width of less than 60px, but there's some slack between that minimum and wide enough to display content without having it cut off by the edge of the viewport. Perhaps it'd be desirable to check before rendering whether there's enough horizontal space to fit the element, and if not, to overlay it on the content in the style of a tooltip or otherwise provide some sort of fallback.

Just a thought, anyway; I noticed and figured I'd mention. It's a nifty little library in general; nice work!

**Comment by fdansv** [comment]
Yes, this is totally a todo. For the sake of not complicating myself too much at the beginning, I just limited rendering whenever there's some real estate for the note to show correctly. A tooltip over the element makes a lot of sense.
